server/app.js

```diff
@@ -1,8 +1,12 @@
 const express = require("express")
 const app = express()
+const endpoints = require("../endpoints.json")
```

Comment:
By convention, data shouldn't be imported into the app.js, as it means you have data existing on the memory of the server all the time, even if it isn't requested.